### PR TITLE
update postgres scaler docs with metricName

### PR DESCRIPTION
**Pull request by ycabrer**
Signed-off-by: ycabrer <43866176+ycabrer@users.noreply.github.com>

Documentation update for PR https://github.com/kedacore/keda/pull/1381

### CODE REVIEW

File: content/docs/1.4/scalers/postgresql.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Finally, a user inserts a query that returns the desired value
 
 - `query:` What query to poll postgresql with. Query must return an integer.
 - `targetQueryValue` - a threshold that is used as `targetAverageValue` in HPA.
+- `metricName` - an optional name to assign to the metric
```
Comment (zroubalik) [inline content/docs/1.4/scalers/postgresql.md:36]: Could you please add more information on the metricName and its usage. ie. that KEDA will generate the metric name from xyz values and this setting could be used to override. And that this is important only if there are multiple postresql scalers in one ScaledObject. Thanks

**Review by zroubalik — CHANGES_REQUESTED** [review]

**Comment by zroubalik** [comment]
could you please target this against 2.1 and not 1.4, eg `content/docs/2.1/scalers/`

**Comment by ycabrer** [comment]
Whoops, that would help. Yes.

**Comment by ycabrer** [comment]
@zroubalik I hope that's better. Let me know if it needs anything else!

Thank you!

File: content/docs/2.1/scalers/postgresql.md [hunk 2]
```diff
@@ -33,8 +33,9 @@ Finally, a user inserts a query that returns the desired value
 
 - `query:` What query to poll postgresql with. Query must return an integer.
 - `targetQueryValue` - a threshold that is used as `targetAverageValue` in HPA.
+- `metricName` - an optional name to assign to the metric. If not set keda will generate a name based on either the connection string if set or the db name. If using more than one trigger it is required that all metricNames be unique.
```
Comment (zroubalik) [inline content/docs/2.1/scalers/postgresql.md:36]:
```suggestion
- `metricName` - an optional name to assign to the metric. If not set KEDA will generate a name based on either the connection string if set or the db name. If using more than one trigger it is required that all metricNames be unique.
```

**Review by zroubalik — COMMENTED** [review]
Looking good, minor nit only!

File: content/docs/2.1/scalers/postgresql.md (hunk 2 above)
Comment (ycabrer) [reply content/docs/2.1/scalers/postgresql.md:36]: I'll get it updated now

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**